spec/mailer/planning_application_mailer_spec.rb

```diff
@@ -797,9 +797,11 @@
       expect(mail_body).to include(
         "This is a copy of your neighbour consultation letter."
       )
-      expect(mail_body).to include(
-        "Submit your comments by #{(1.business_day.from_now + 21.days).to_date.to_fs(:day_month_year)}"
-      )
+      travel_to(Time.zone.local(Time.zone.now.year, Time.zone.now.month, Time.zone.now.day, 16, 0, 0)) do
+        expect(mail_body).to include(
+          "Submit your comments by #{(1.business_day.from_now + 21.days).to_date.to_fs(:day_month_year)}"
+        )
+      end
```

Comment:
Why not just `travel_to` with a specific date/time rather than using the current date?

If you want to keep using the current date, you can use `change` to simplify it, e.g.

``` ruby
travel_to(Time.zone.now.change(hour: 16)) do
  # do stuff
end
```